Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -5678,6 +5678,54 @@ script:
     - contextPath: MicrosoftATP.MachineAlerts.MachineID
       description: The alerts machine ID.
       type: String
+  - arguments:
+    - default: false
+      description: The machine ID.
```

Comment:
From where can the user get this ID? Another command output?
Add to the description.